Podfile

```diff
@@ -4,7 +4,7 @@ workspace 'VideoQuickStart'
 platform :ios, '9.0'
 
 abstract_target 'TwilioVideo' do
-  pod 'TwilioVideo', '~> 2.5'
+  pod 'TwilioVideo', '~> 2.6.0-preview1'
```

Comment:
2.6.0-preview2

Reply:
Dang it.. Copy/Paste error. lol.

I had to start with preview1 so I could at least have it have most of the header file links that CocoaPods does, and then copied the RC in manually. Nice catch.